certhandler/modules/pkcs11module/key.go

```diff
@@ -138,7 +149,8 @@ func (key *pkcs11PrivateKeyRSA) Sign(rand io.Reader, digest []byte, opts crypto.
 	return signature, nil
 }
 
-func (key *pkcs11PrivateKeyECC) Sign(rand io.Reader, digest []byte, opts crypto.SignerOpts) (signature []byte, err error) {
+func (key *pkcs11PrivateKeyECC) Sign(
```

Comment:
Make the same formatting as in above function.

Reply:
done

Reply:
done